.github/workflows/tests_docker.yml

```diff
@@ -54,6 +54,8 @@ jobs:
       run: docker exec playwright-docker-${{ matrix.tag }}-test npm ci
     - name: Run "npm run build" inside docker
       run: docker exec playwright-docker-${{ matrix.tag }}-test npm run build
+    - name: Test installing Chrome&MSEdge
+      run: docker exec playwright-docker-${{ matrix.tag }}-test npx playwright install chrome msedge
```

Comment:
It might make sense to do it in a separate instance to not affect other tests below.